### ISSUE TITLE
YubiKey Manager 1.1.4 doesn't work on Windows 10 20H1

### ISSUE THREAD

**Issue by rallenh** [Yubico/yubikey-manager-qt#241]
I have installed YubiKey Manager 1.1.4. I am running it as Administrator.  Windows identifies my YubiKey(s) in Devices and Printers. They are also in the System Tray (to Eject). I hear the Windows USB sounds when I insert and remove my YubiKey(s). The Secret Key NFC key(s) work fine with AWS Console via Opera.

The YubiKey Manager never says anything but "Insert your YubkKey". Sometimes, when I click the "X" in the upper-right dialog box, it also doesn't exit completely, the GUI is gone but ykman-gui.exe is still running and I have to kill it manually.

I have 5 new NFC keys and none of them work with YubiKey Manager.

```
> ver
Microsoft Windows [Version 10.0.19041.388]
```

```
> ykman.exe --log-level DEBUG info
2020-07-27T02:23:27-0400 INFO [ykman.logging_setup.setup:58] Initialized logging for ykman version: 3.1.1
Usage: ykman.exe [OPTIONS] COMMAND [ARGS]...
Try "ykman.exe -h" for help.

Error: No YubiKey detected!
```

```
> ykman.exe --log-level DEBUG list --serials
2020-07-27T02:23:40-0400 INFO [ykman.logging_setup.setup:58] Initialized logging for ykman version: 3.1.1
2020-07-27T02:23:42-0400 DEBUG [ykman.device.__init__:198] Read config from device...
2020-07-27T02:23:42-0400 DEBUG [ykman.driver_ccid.send_apdu:228] SEND: b'00a4040008a000000527471117'
2020-07-27T02:23:42-0400 DEBUG [ykman.driver_ccid.send_apdu:230] RECV: b'5669727475616c206d6772202d2046572076657273696f6e20352e322e349000'
2020-07-27T02:23:42-0400 DEBUG [ykman.driver_ccid.send_apdu:228] SEND: b'001d000000'
2020-07-27T02:23:42-0400 DEBUG [ykman.driver_ccid.send_apdu:230] RECV: b'2e0102023f0302023f020400bac13704010105030502040602000007010f0801000d02023f0e02023f0a01000f01009000'
2020-07-27T02:23:42-0400 DEBUG [ykman.device.__init__:200] Success!
2020-07-27T02:23:42-0400 DEBUG [ykman.device.__init__:289] Identified YubiKey 5
2020-07-27T02:23:42-0400 DEBUG [ykman.driver_ccid.close:269] Close <ykman.driver_ccid.CCIDDriver object at 0x00000232EC07CCD0>
2020-07-27T02:23:42-0400 DEBUG [ykman.driver_ccid.__del__:273] Destroy <ykman.driver_ccid.CCIDDriver object at 0x00000232EC07CCD0>
2020-07-27T02:23:42-0400 DEBUG [ykman.driver_ccid.close:269] Close <ykman.driver_ccid.CCIDDriver object at 0x00000232EC07CCD0>
```

**Comment by fdennis** [comment]
Hi,
Could you try running `ykman.exe -v` and paste the result here?

**Comment by rallenh** [comment]
Hi @fdennis,

```
> ykman.exe --log-level DEBUG --version
YubiKey Manager (ykman) version: 3.1.1
Libraries:
    libykpers 1.20.0
    libusb 1.0.22
```

The old Yubico Personalization Tool (v3.1.25) works:
![image](https://user-images.githubusercontent.com/23007132/88517853-811ab080-cfbd-11ea-90d6-e98a27127012.png)

**Comment by fdennis** [comment]
It looks like you have everything to make it work. Can you try to uninstall and then reinstalling it? If that does not work, do you get the same issue with 1.1.3?

I am sitting on a Windows 10 build 19041.388 and 1.1.4 works for me.

**Comment by rallenh** [comment]
I got an MSI pending reboot error during the WDK install (I was trying to get DevCon.exe installed). So I rebooted and viola! I didn't get a reboot prompt from your installer. And I uninstalled/re-installed quite a few times for 1.1.3 and 1.1.4.

I'm good, thanks!
![image](https://user-images.githubusercontent.com/23007132/88609268-b07afd00-d051-11ea-823f-a1e053213d95.png)
![image](https://user-images.githubusercontent.com/23007132/88609270-b375ed80-d051-11ea-82a8-ce942421091a.png)
